torch/distributions/binomial.py

```diff
@@ -94,3 +94,17 @@ def enumerate_support(self):
         values = values.view((-1,) + (1,) * len(self._batch_shape))
         values = values.expand((-1,) + self._batch_shape)
         return values
+
+    def entropy(self):
```

Comment:
It looks like your default implementation of `ExponentialFamily.entropy()` already correctly raises `NotImplementedError` without performing any extra computation. Do you have a reason for adding this method here?

Reply:
Oh, I have no idea why that went there. I will remove it right away. Thanks!!